clients/cobol-lsp-vscode-extension/src/__tests__/services/SettingsTest.spec.ts

```diff
@@ -14,27 +14,21 @@
 import * as fs from "fs-extra";
 import * as path from "path";
 import * as vscode from "vscode";
-import { C4Z_FOLDER, GITIGNORE_FILE } from "../../constants";
-import {
-  createFileWithGivenPath,
-  SettingsService,
-} from "../../services/Settings";
+import { GITIGNORE_FILE } from "../../constants";
+import { SettingsService } from "../../services/Settings";
 import { SettingsUtils } from "../../services/util/SettingsUtils";
 
 const fsPath = "tmp-ws";
 let wsPath: string;
-let c4zPath: string;
 let filePath: string;
```

Comment:
Both wsPath and filePath is not used anymore.

Reply:
removed : c5dae9b691634ef09c7e78882e44c4ffb6c58a43